### ISSUE TITLE
deprecation messages leave extra lines in compiler output

### ISSUE THREAD

**Issue by Y-Less** [Zeex/pawn#187]
I have this:

```pawn
#pragma deprecated Use formatex instead
forward va_formatex(output[], size = sizeof(output), const fmat[], va_:STATIC_ARGS);
```

Which gives the following output when the function is used:

```
D:\work\SA-MP\gamemodes\deprecated.pwn(42) : warning 234: function is deprecated (symbol "va_formatex") Use formatex instead

D:\work\SA-MP\gamemodes\deprecated.pwn(53) : warning 234: function is deprecated (symbol "va_formatex") Use formatex instead

```

There are blank lines placed between the error lines.  I've tried a few ways to get rid of them - changing line endings and escaping them, but neither work.  With no message, the spacing is correct:

```pawn
#pragma deprecated
forward va_formatex(output[], size = sizeof(output), const fmat[], va_:STATIC_ARGS);
```

Which gives the following output when the function is used:

```
D:\work\SA-MP\gamemodes\deprecated.pwn(42) : warning 234: function is deprecated (symbol "va_formatex")
D:\work\SA-MP\gamemodes\deprecated.pwn(53) : warning 234: function is deprecated (symbol "va_formatex")
```

**Comment by Dayvison** [comment]
same for  `#warning`

**Comment by YashasSamaga** [comment]
After reading the warning/pragma deprecated directive, the compiler skips white spaces (or until it encounters a null character). It passes the remaining line string as it is to the error function. This line string would include a new line character which was added by the scripter.

However, the compiler adds also adds a new line by itself at the end of every warning and pragma deprecated. Therefore, the compiler would print two new lines.

**Comment by Zeex** [comment]
This was fixed by #192 and https://github.com/Zeex/pawn/commit/e7ecf386f8e82ed03a3627712d2064940c61aeb4.
